src/components/Notifications/TimeConfig.tsx

```diff
@@ -1,22 +1,36 @@
-import { Card, CardBody, Dropdown, DropdownItem, DropdownToggle, HelperText, HelperTextItem, PageSection, Radio, Split, SplitItem, Stack, StackItem,
-    Text, TextVariants, TimePicker, Title } from '@patternfly/react-core';
-import React from 'react';
+import { Card, CardBody, Dropdown, DropdownItem, DropdownToggle, HelperText, HelperTextItem, PageSection,
+    Radio, Skeleton, Split, SplitItem, Stack, StackItem, Text, TextVariants, TimePicker, Title } from '@patternfly/react-core';
+import React, { useMemo } from 'react';
 import timezones from 'timezones.json';
 import { style } from 'typestyle';
 
+import { useGetTimePreference } from '../../services/Notifications/GetTimePreference';
 import { useUpdateTimePreference } from '../../services/Notifications/SaveTimePreference';
 
 const dropDownClassName = style({
     width: '280px'
 });
 
+const timePrefSkeleton = <Skeleton />;
+
 export const TimeConfigComponent: React.FunctionComponent = () => {
 
     const [ radioSelect, setRadioSelect ] = React.useState(false);
     const [ showCustomSelect, setShowCustomSelect ] = React.useState(false);
 
+    const getTimePreference = useGetTimePreference();
     const saveTimePreference = useUpdateTimePreference();
 
+    const timePref = useMemo(() => {
+        if (getTimePreference.payload?.status === 200) {
+            return getTimePreference.payload.value;
+        }
+
+        if (getTimePreference.payload?.status === undefined) {
+            return timePrefSkeleton && undefined;
```

Comment:
:thinking: Why do you need to do the `&& undefined` ? 

I think you could simply this with only:

```suggestion
            return <Skeleton />;
```

and by removing the other variable.

Reply:
If the issue is that some of the other components require a string, you could simply return undefined here and do elsewhere:

`timePref ? <TheOtherComponent> : <Skeleton />`